ui/imports/utils/Utils.qml

```diff
@@ -614,6 +614,20 @@ QtObject {
             return 34
     }
 
+    function getTimerString(timeInSecs) {
+        let result = ""
+        let hour = Math.floor(timeInSecs/60/60)
+        let mins = Math.floor(timeInSecs/60)
+        let secs = Math.floor(timeInSecs%60)
+        if(hour > 0 )
+            result += " " + hour + " " + (hour > 1 ? qsTr("hours") : qsTr("hour"))
```

Comment:
@Khushboo-dev-cpp you could use translation for plurals here, check this https://doc.qt.io/qt-5/i18n-source-translation.html#handling-plurals

Reply:
Definitely :police_officer:

Reply:
In a nutshell, `result += qsTr("%n hour(s)", "", hour)`, and so on...

Reply:
And we have a nice collection of these functions in `LocaleUtils.qml` already ;) Hint hint